src/renderer/entities/transaction/lib/transactionBuilder.ts

```diff
@@ -412,6 +413,27 @@ function buildVote({ chain, accountId, referendumId, trackId, vote }: VoteParams
   };
 }
 
+type RevoteParams = {
+  chain: Chain;
+  accountId: AccountId;
+  trackId: TrackId;
+  referendumId: ReferendumId;
+  vote: TransactionVote;
+};
+
+function buildRevote({ chain, accountId, referendumId, trackId, vote }: RevoteParams): RevoteTransaction {
+  return {
+    chainId: chain.chainId,
+    address: toAddress(accountId, { prefix: chain.addressPrefix }),
+    type: TransactionType.REVOTE,
+    args: {
+      track: trackId,
```

Comment:
Do we really need track here? You remove `removeVote` which use track from extrinsic for revote